.github/workflows/windows.yml

```diff
@@ -78,6 +78,8 @@ jobs:
           scons --version
 
       # We should always be explicit with our flags usage here since it's gonna be sure to always set those flags
+      # TODO We'd like to use `warnings=extra` with MSVC but Godot doesn't compile yet with that level of warnings.
+      # See https://github.com/godotengine/godot/issues/66537
```

Comment:
That would answer my previous question: extra is including more than "all" as extra=-W4 and all=-W3.